src/Components/TodoList/ListItem/ListItem.tsx

```diff
@@ -1,9 +1,29 @@
-import React from 'react'
+import { Box, Button, Checkbox, Typography } from "@mui/material";
+import React from "react";
+import { COLORS } from "../../../colors";
 
-export default function ListItem() {
+const removeButtonSx = {
+  bgcolor: COLORS.RED_BUTTON,
+  textTransform: "none",
+  "&:hover": {
+    backgroundColor: COLORS.RED_BUTTON_HOVER,
+  },
+};
+
+interface ListItemProps {
+  description: string;
+}
+
+export default function ListItem({ description }: ListItemProps) {
   return (
-    <div>
-      list item
-    </div>
-  )
+    <Box sx={{ display: "flex", flex: 1, alignItems: "center" }}>
+      <Checkbox />
+      <Typography sx={{ display: "flex", width: "100%" }}>
+        {description}
+      </Typography>
+      <Button variant="contained" sx={removeButtonSx}>
```

Comment:
And here, please use the additional styles like this:

sx={{...removeButtonSx, ...removeButtonExtraSx}}